packages/local_auth/lib/auth_strings.dart

```diff
@@ -68,20 +68,23 @@ class IOSAuthMessages {
     this.goToSettingsButton,
     this.goToSettingsDescription,
     this.cancelButton,
+    this.localizedFallbackTitle,
   });
 
   final String? lockOut;
   final String? goToSettingsButton;
   final String? goToSettingsDescription;
   final String? cancelButton;
+  final String? localizedFallbackTitle;
 
-  Map<String, String> get args {
-    return <String, String>{
+  Map<String, String?> get args {
+    return <String, String?>{
```

Comment:
You can conditionally include `'localizedFallbackTitle'` rather than making the values nullable. (Which will require the change to handle `nil` mentioned above.)